cpp/perspective/CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ elseif(PSP_PYTHON_BUILD)
     if(NOT DEFINED PSP_PYTHON_VERSION)
         set(PSP_PYTHON_VERSION 3.10)
     endif()
-    if($ENV{PYODIDE})
+    if(PSP_WASM_BUILD)
         set(PSP_PYODIDE 1)
```

Comment:
`PSP_PYODIDE` should be an `option` not a random variable, as in https://github.com/finos/perspective/pull/2677

Reply:
I'm not sure it makes sense to have pyodide references at all anymore. We're really just building an emscripten dynamic library that can be dynamically loaded by any compatible emscripten compiled Python implementation. I think we should just delete references to pyodide all together.

Reply:
sounds good to me, in #2677 i separate out emscripten in the top platform block, we could just rely on that